includes/class-newspack-popups-model.php

```diff
@@ -1279,8 +1279,9 @@ public static function foreground_color_for_background( $background_color ) {
 	 * @return string Inline styles attribute.
 	 */
 	public static function container_style( $popup ) {
-		$background_color = $popup['options']['background_color'];
-		$foreground_color = self::foreground_color_for_background( $background_color );
+		$hide_border      = $popup['options']['hide_border'];
+		$background_color = $hide_border ? 'transparent' : $popup['options']['background_color'];
```

Comment:
With a transparent background, an overlay prompt won't look great:

<img width="386" alt="image" src="https://user-images.githubusercontent.com/7383192/176634160-a366ebdc-8c63-4d35-b3d2-971b95db8feb.png">

Reply:
Ah, my test prompt happened to have a Group block with a background. 🤦 [8621638](https://github.com/Automattic/newspack-popups/pull/897/commits/86216384223d86e768f1ae7277927d71d2d0cc40) should make prompts look better without that.